net/packet/tsmp.go

```diff
@@ -39,6 +39,16 @@ type TailscaleRejectedHeader struct {
 	Dst    netaddr.IPPort        // rejected flow's dst
 	Proto  IPProto               // proto that was rejected (TCP or UDP)
 	Reason TailscaleRejectReason // why the connection was rejected
+
+	// MaybeBroken is whether the rejection is non-terminal (the
+	// client should not fail immediately). This is sent by a
+	// target when it's not sure whether it's totally broken, but
+	// it might be.  For example, the target tailscaled might
+	// think its host firewall or IP forwarding aren't configured
+	// properly, but they might be. When true, the message is
+	// simply an FYI as a potential reason to use for later when
+	// the pendopen connection tracking timer expires.
+	MaybeBroken bool
```

Comment:
Seeing this used in the code makes me want to call it `OnlyMaybeBroken` or `MaybeOK`.

Reply:
And in other places `MaybeBroken` is better. Meh.